app/assets/javascripts/admin/utils/services/status_message.js.coffee

```diff
@@ -1,11 +1,11 @@
-angular.module("admin.utils").factory "StatusMessage", ($timeout) ->
+angular.module("admin.utils").factory "StatusMessage", ->
   new class StatusMessage
     types:
-      progress: {timeout: false, style: {color: '#ff9906'}}
-      alert:    {timeout: 5000,  style: {color: 'grey'}}
-      notice:   {timeout: false, style: {color: 'grey'}}
-      success:  {timeout: 5000,  style: {color: '#9fc820'}}
-      failure:  {timeout: false, style: {color: '#da5354'}}
+      progress: {style: {color: '#ff9906'}}
+      alert:    {style: {color: 'grey'}}
+      notice:   {style: {color: 'grey'}}
+      success:  {style: {color: '#9fc820'}}
```

Comment:
I wonder if the success message should not disappear automatically anyway. No ?

Reply:
I'm not sure what you are saying. Are you agreeing that no message should disappear? Or are you saying that success messages should disappear?

Reply:
Sorry, I wasn't clear, because I missed a point: you've been talking about success messages too. I thought you talked, on the slack thread, only about the failure/error message.
My bad ;)